### ISSUE TITLE
Docker Image

### ISSUE THREAD

**Issue by zachriggle** [RPISEC/MBE#1]
It'd be really great if `setup_image.sh` could be run by one of the RPI members, and packaged as a Docker image.  This would make it easier for other users to at least get it up and running :)

**Comment by RPISEC** [comment]
We provided a VM image in the initial release. See the [release](https://github.com/RPISEC/MBE/releases
https://github.com/RPISEC/MBE/releases/download/v1.0_release/MBE_VM.vmdk.gz

**Comment by gaasedelen** [comment]
Reopened because I think this is worth exploring.

**Comment by clarkb7** [comment]
@Manouchehri: No. The only kernel stuff is for lab 10, which isn't included in this image.

**Comment by Manouchehri** [comment]
Thanks, should make it easier.
